resources/views/layouts/app.blade.php

```diff
@@ -22,7 +21,6 @@
 
     <!-- Icons -->
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
```

Comment:
kan toch ook weg?

Reply:
We gebruiken fontawesome icons bij user CRUD dus, kan best blijven